docs/endpoints/constructorStandings.md

```diff
@@ -0,0 +1,205 @@
+[← Documentation Home](/docs/README.md)
+# Constructor Standings
+
+Gets a season's constructors standings from first to last place.
+
+**URL** : `/ergast/f1/{season}/constructorstandings/`
+
+[Available Query Parameters](./README.md#query-parameters)
+
+---
+
+## Route Parameters:
+
+### Season \*\*REQUIRED\*\*
+
+**Filters for the constructors standing of a specified season. Year numbers are valid as is `current` to get the current seasons constructors standings**
+
+`/{season}/` -> ex: `/ergast/f1/2024/constructorstandings/`
+
+**Note**: To utilize the `season` parameter it needs to be the first route after `/ergast/f1/`
+
+---
+
+### Round
+
+**Filters for the constructors standings after a specified round in a specific season. Round numbers 1 -> n races are valid as well as `last`**
+
+`/{season}/{round}/` -> ex: `/ergast/f1/2024/5/constructorstandings/`
+
+**Note**: To utilize the `round` parameter it must be combined with a season filter and needs to be the first route after `/ergast/f1/{season}/`
+
+---
+
+### constructors
+
+**Filters for only for a specific constructors's constructors standing information for a given year**
+
+`/constructors/{constructorsId}/` -> ex: `/ergast/f1/2024/constructors/ferrari/constructorstandings/`
+
+---
+
+### position
+
+**Filters for only the constructor in a given position in a given year**
+
+`/{finishPosition}` -> ex: `/ergast/f1/2024/constructorstandings/1/`
+
+**Note**: The position must be at the end after any filters and after `/constructorstandings/`
+
+---
+
+## Success Response
+
+**Code** : `200 OK`
+
+**Response Fields** :
+
+[Common Response Fields](./README.md#common-response-fields)
+
+`MRData.StandingsTable` : The object containing the season's constructors standing information.
+
+`MRData.StandingsTable.season` : The filtered season.
+
+`MRData.StandingsTable.round` : The round that the season the standings represent.
+
+`MRData.StandingsTable.StandingsLists` : The list of constructors standings.
+
+`MRData.StandingsTable.StandingsLists[i]` : A given constructors standings list object.
+
+`MRData.StandingsTable.StandingsLists[i].ConstructorStandings` : The list of constructors standings objects.
+
+---
+
+## Constrcutors Standing Object Fields:
+
+|Field|Required|
+|---|:---:|
+|position| ✅ |
+|positonText|✅|
+|points|✅|
+|wins|✅|
+|Constructor|✅|
```

Comment:
```suggestion
|position| ❌ |
|positonText|✅|
|points|✅|
|wins|✅|
|Constructor|✅|
Possible values for positionText include: `E` Excluded (2007 McLaren), `D` Disqualified, `-` for ineligible or the position as a string otherwise.  
```